lib/App/index.js

```diff
@@ -2040,6 +2121,31 @@ class App {
       }
     }
 
+
+    /*
+    * If typescript is used, then add the types for node and add a .tsconfig.json file 
+    */
+    if (answers.typescript) {
+      await NpmCommands.install({ save: true }, {
+        id: '@types/node'
+      });
+
+      const tsConfigJson = {
+        "compilerOptions": {
+            "module": "commonjs",
+            "target": "ES6",
+            "outDir": "ts-output",
+            "rootDir": "",
+            "moduleResolution": "node",
+            "esModuleInterop": true,
+            "removeComments": true,
+            "sourceMap": true,
```

Comment:
Does this work as expected? I think node 12 does not support proper source map support out of the box, you can test this by throwing an error somewhere in your code to check if the stacktrace is refering to .js or .ts files.